app/assets/stylesheets/components/_banner.scss

```diff
@@ -0,0 +1,20 @@
+.banner {
+  background-size: cover;
+  background-position: center;
+  padding: 150px 0;
+}
```

Comment:
You  can nest the h1 and p inside the banner css if you want to be more railsy :) but stet is OK too